source/help/apps/desktop-changelog.rst

```diff
@@ -0,0 +1,485 @@
+Desktop Application Changelog
+=============================
+
+Release v3.4.1
+-----------------------------
+
+Release date: September 30, 2016
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
+
+Version number updated to 3.4 to make numbering consistent with Mattermost server and mobile app releases. This change will not imply bi-monthly releases.
+
+-  v3.4.1, released 2016-09-30
+
+   -  (OS X) Fixed an issue where the app window pops up second to foreground when a new message is received
+
+-  v3.4.0, released 2016-09-22
+
+   -  Original v3.4 release
+
+Improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  Current team and channel name shown in window title bar
+-  Team tab is bolded for unread messages and has a red dot with a count of unread mentions
+-  Added an option to configure whether a red badge is shown on taskbar icon for unread messages
+-  Added new shortcuts:
+
+   -  CTRL+S: sets focus on the Mattermost search box
+   -  ALT+Left Arrow: go to previous page in history
```

Comment:
Although I do have two questions to clarify:
 - should the keyboard buttons be bolded though? Or not?
 - should "Left Arrow" be in all capital letters? Given it's not the keyboard button name, like ENTER and CTRL, I think it shouldn't be